certified-connectors/Unearth Ingest/apiDefinition.swagger.json

```diff
@@ -0,0 +1,293 @@
+				"responses": {
+					"200": {
+						"description": "Success"
+					}
+				}
+			}
+		},
+		"/api/ProcessingStatus": {
+			"get": {
+				"tags": ["Status"],
+				"summary": "Gets the processing status of an operation",
+				"description": "Gets the processing status of an operation",
+				"operationId": "Status-GetProcessingStatus",
+				"produces": ["text/plain", "application/json"],
+				"parameters": [
+					{
+						"description": "The correlation Id of the item",
+						"x-ms-summary": "The correlation Id of the item",
+						"in": "query",
+						"name": "id",
+						"type": "string"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "Success"
+					},
+					"202": {
+						"description": "Accepted"
+					},
+					"400": {
+						"description": "Bad Request",
+						"schema": {
+							"$ref": "#/definitions/ProblemDetails"
+						}
+					},
+					"524": {
+						"description": "Processing of the Document failed with given error"
+					}
+				}
+			}
+		},
+		"/api/OperationStatus": {
+			"get": {
+				"tags": ["Status"],
+				"summary": "Gets the Operation status of an operation",
+				"description": "Gets the Operation status of an operation",
+				"operationId": "Status-GetOperationStatus",
+				"produces": ["text/plain", "application/json"],
+				"parameters": [
+					{
+						"description": "The correlation Id of the item",
+						"x-ms-summary": "The correlation Id of the item",
+						"in": "query",
+						"name": "id",
+						"type": "string"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "Success"
+					},
+					"202": {
+						"description": "Accepted"
+					},
+					"400": {
+						"description": "Bad Request",
+						"schema": {
+							"$ref": "#/definitions/ProblemDetails"
+						}
+					},
+					"524": {
+						"description": "Processing of the Document failed with given error"
+					}
+				}
+			}
+		}
+	},
+	"definitions": {
+		"ProblemDetails": {
+			"type": "object",
+			"properties": {
+				"type": {
+					"type": "string"
+				},
+				"title": {
+					"type": "string"
+				},
+				"status": {
+					"format": "int32",
+					"type": "integer"
+				},
+				"detail": {
+					"type": "string"
+				},
+				"instance": {
+					"type": "string"
+				}
+			},
+			"additionalProperties": {}
+		},
+		"StatusDetails": {
+			"type": "object",
+			"properties": {
+				"code": {
+					"format": "int32",
+					"type": "integer"
+				},
+				"status": {
+					"type": "string"
+				}
+			},
+			"additionalProperties": {}
+		}
+	},
+	"parameters": {},
+	"responses": {},
+	"securityDefinitions": {
+		"oauth2_auth": {
+			"type": "oauth2",
+			"flow": "accessCode",
+			"authorizationUrl": "https://login.windows.net/common/oauth2/authorize",
+			"tokenUrl": "https://login.windows.net/common/oauth2/authorize",
+			"scopes": {
+				"api://275fb2fd-9e15-4a1b-9c7e-2f0f7feb95b1/processapi": "api://275fb2fd-9e15-4a1b-9c7e-2f0f7feb95b1/processapi"
+			}
+		}
+	},
+	"security": [
```

Comment:
@J05HM0N5TER would it be possible to remove `securityDefinitions` and `security` properties from this file? The scopes contain secret-look-a-like value and we may receive security alerts because of this.